src/Xref.ts

```diff
@@ -198,6 +203,33 @@ function buildFigureLink(
   }
 }
 
+let decimalBullet = Array.from({ length: 100 }).map((a, i) => '' + (i + 1));
+let alphaBullet = Array.from({ length: 26 }).map((a, i) =>
+  String.fromCharCode('a'.charCodeAt(0) + i)
+);
+// prettier-ignore
+let romanBullet = ['i', 'ii', 'iii', 'iv', 'v', 'vi', 'vii', 'viii', 'ix', 'x', 'xi', 'xii', 'xiii', 'xiv', 'xv', 'xvi', 'xvii', 'xviii', 'xix', 'xx', 'xxi', 'xxii', 'xxiii', 'xxiv', 'xxv'];
+let bullets = [decimalBullet, alphaBullet, romanBullet, decimalBullet, alphaBullet, romanBullet];
```

Comment:
Why do we have to use stylised step numbers here? Regular Indic/Arabic numerals should do just fine, right? I hate all this complexity and I hate that it's coupled with the styles.

Reply:
Personally I find "step 6.a.vi" a lot nicer than "step 6.1.3". I'm not dead set against changing it, but I don't think it should be changed as part of this PR; feel free to bring it up on the editor call.

Reply:
Oh I was thinking these were for internal references for some reason. I see now that it's for the text in the reference link. Yeah I guess this is fine for now. I wish we could somehow coordinate this logic with the CSS styling.